docs/standards.md

```diff
@@ -0,0 +1,53 @@
+# Mapping Standards
+
+This document details the standards that all contributions to Parchment mappings must follow.
+
+### Parameter Names
+
+1. Use American English for spellings
+    - Example: `color`, not `colour`, `armor`, not `armour`
+1. All names should be prefixed with `p`
+    - Example: `pLevel`, `pXPos`
+1. Names should make sense without the prefix, and should be valid java identifiers NOT reserved keywords as such:
+    - Example: `pFloat` is NOT valid, as `float` is a reserved keyword.
+    - Example: `pOs` is not a correct use of the prefix, it should be `pPos`
+1. Parameter names may be named based on the types, and the context in which they are used. They should be verbose and use complete words - do not omit essential information for brevity's sake.
+    - Example: `BlockPos pAdjacentPos, BlockPos pCurrentPos`, not `BlockPos pPos1, BlockPos pPos2`
+1. Avoid single letters, or abbreviations.
+    - Example: `pNorthWest` over `pNW`, `pMatrixStack` over `pMStack`. 
+    - Exception: common abbreviations can be used: `IO` / `NBT`, etc.
+    - Exception: common class / parameter names can be shortened: `BlockPos pPos, BlockState pState, WorldGenLevel pLevel`
+1. Use Lower Camel Case, treating the `p` prefix as the first word.
+    - Example: `pLowerCamelCase`
+3. Do not use `$` or `_` in variable names
+4. All parameters should match the regex `p[A-Z][A-Za-z0-9]*`.
+    - Simply, parameters should contain only alphanumeric characters, and start with the `p` prefix, followed by a capital letter.
+    - Example: `pDot`, `pPos`, `pBlockState`
```

Comment:
Would've been good to also add an example of a name with a number character, but this is fine as-is.